dev/Repeater/TestUI/RepeaterTestUIPage.xaml

```diff
@@ -5,89 +5,87 @@
     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
     xmlns:controls="using:Microsoft.UI.Xaml.Controls">
 
-    <Grid HorizontalAlignment="Center" VerticalAlignment="Center">
-        <Grid.Resources>
+    <controls:LayoutPanel  Background="{ThemeResource ApplicationPageBackgroundThemeBrush}" HorizontalAlignment="Center" VerticalAlignment="Center">
+        <controls:LayoutPanel.Layout>
+            <controls:FlowLayout Orientation="Vertical" MinColumnSpacing="5" MinRowSpacing="5"/>
+        </controls:LayoutPanel.Layout>
+
+        <controls:LayoutPanel.Resources>
             <controls:FlowLayout x:Key="flow" MinRowSpacing="5" MinColumnSpacing="5" />
             <controls:StackLayout x:Key="stack"  Spacing="5" />
-        </StackPanel.Resources>
+        </controls:LayoutPanel.Resources>
 
-        <controls:LayoutPanel >
-            <controls:LayoutPanel.Layout>
-                <controls:FlowLayout Orientation="Vertical" MinColumnSpacing="5" MinRowSpacing="5"/>
-            </controls:LayoutPanel.Layout>
+        <StackPanel>
+            <TextBlock Text="ItemsSource Samples" Style="{ThemeResource HeaderTextBlockStyle}"/>
```

Comment:
nit: You could also use the `StandardGroupHeader` style that was introduced with #1793, but this looks fine to me.